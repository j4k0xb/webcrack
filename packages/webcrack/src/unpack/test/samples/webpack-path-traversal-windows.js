(function (e) {
  var n = {};
  function o(r) {
    if (n[r]) {
      return n[r].exports;
    }
    var a = (n[r] = {
      i: r,
      l: false,
      exports: {},
    });
    e[r].call(a.exports, a, a.exports, o);
    a.l = true;
    return a.exports;
  }
  o.p = '';
  o((o.s = 386));
})({
  './\\..\\node_modules\\debug\\src\\index': function (e, t, n) {},
});
