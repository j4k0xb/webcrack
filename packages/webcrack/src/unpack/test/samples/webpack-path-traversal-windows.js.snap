WebpackBundle {
  "entryId": "386",
  "modules": Map {
    "./\..\node_modules\debug\src\index" => WebpackModule {
      "ast": ,
      "id": "./\..\node_modules\debug\src\index",
      "isEntry": false,
      "path": "././\..\node_modules\debug\src\index.js",
    },
  },
  "type": "webpack",
}